### PR TITLE
fix: dynamic import chunk is_entry should be false

### PR DESCRIPTION
**Pull request by underfin**
### Test Plan

---

### CODE REVIEW

**Comment by underfin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #360** <a href="https://app.graphite.dev/github/pr/rolldown-rs/rolldown/360?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #361** <a href="https://app.graphite.dev/github/pr/rolldown-rs/rolldown/361?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
      * **PR #362** <a href="https://app.graphite.dev/github/pr/rolldown-rs/rolldown/362?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #363** <a href="https://app.graphite.dev/github/pr/rolldown-rs/rolldown/363?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/rolldown-rs/rolldown/361?utm_source=stack-comment).

File: crates/rolldown/tests/common/case.rs [hunk 1]
```diff
@@ -78,6 +78,30 @@ impl Case {
       .collect::<Vec<_>>()
       .join("\n");
     self.snapshot.append(artifacts);
+
+    if self.fixture.test_config().snapshot_output_stats {
+      self.render_stats_to_snapshot(assets);
+    }
+  }
+
+  fn render_stats_to_snapshot(&mut self, assets: Vec<Output>) {
```
Comment (hyf0) [inline crates/rolldown/tests/common/case.rs]: Cool. It would be good if you could separate the testing changes in another PR. 

I'm not sure if we need this information. I tend to not add this.

Comment (underfin) [reply crates/rolldown/tests/common/case.rs]: I want to make sure the output stats are tested, Add this to the rust side because it is easy to debugger than js side. Maybe you have another idea for this?

Comment (underfin) [reply crates/rolldown/tests/common/case.rs]: > I'm not sure if we need this information. I tend to not add this.

The information is used at vite, it is also rust exported stats data, it should be tested.

Comment (hyf0) [reply crates/rolldown/tests/common/case.rs]: Any ideas on how rollup or esbuild test this?

Comment (underfin) [reply crates/rolldown/tests/common/case.rs]: You can see here https://github.com/rollup/rollup/blob/master/test/hooks/index.js#L915

Comment (underfin) [reply crates/rolldown/tests/common/case.rs]: I removed the test at the pr, it is can merged at now

**Review by hyf0 — APPROVED** [review]
